setup.py

```diff
@@ -7,7 +7,7 @@ def readme():
 setup(
   name = 'knoema',
   packages = ['knoema'],
-  version = '1.2.52',
+  version = '1.3.53',

```

Comment:
version should be 1.3.0

Reply:
it's a build number. It always should be incremented.

Reply:
When changing a major or minor version, the build number must be reset. Examples: https://pypi.org/project/knoema/#history, https://pypi.org/project/pandas/#history